### PR TITLE
Add url and headers to timeout errors

### PR DESCRIPTION
**Pull request by JonasHallFnx**

### CODE REVIEW

File: client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java [hunk 1]
```diff
@@ -1140,6 +1142,26 @@ public void shouldHandleUnsafeSecureOnUntrustedHost() throws URISyntaxException
...
+        try {
+            TestResource resource = getHttpProxy(server.getServerPort());
+            HttpClient.setTimeout(resource, 10, TimeUnit.MILLISECONDS);
+            resource.servertest("mode").toBlocking().single();
+            fail("expected timeout");
+        } catch (Exception e) {
+            OutputStream baos = new ByteArrayOutputStream();
+            PrintStream stream = new PrintStream(baos, true);
+            e.printStackTrace(stream);
+            assertThat(baos.toString()).contains("/servertest/mode");
```
Comment (splitfeed) [inline client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java]: Couldn't we assert that the entire message "Timeout after %d ms calling %s" is correct?

File: client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java [hunk 2]
```diff
@@ -1140,6 +1142,26 @@ public void shouldHandleUnsafeSecureOnUntrustedHost() throws URISyntaxException
...
+
+        try {
+            TestResource resource = getHttpProxy(server.getServerPort());
+            HttpClient.setTimeout(resource, 10, TimeUnit.MILLISECONDS);
+            resource.servertest("mode").toBlocking().single();
+            fail("expected timeout");
+        } catch (Exception e) {
+            OutputStream baos = new ByteArrayOutputStream();
+            PrintStream stream = new PrintStream(baos, true);
+            e.printStackTrace(stream);
```
Comment (splitfeed) [inline client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java]: Forgotten debug? :)

**Comment by splitfeed** [comment]
Nice 👍

File: client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java (hunk 2 above)
Comment (JonasHallFnx) [reply client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java]: no, it writes to the stream that is used in assert below

File: client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java (hunk 1 above)
Comment (JonasHallFnx) [reply client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java]: sure :-)

File: client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java (hunk 2 above)
Comment (splitfeed) [reply client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java]: Ah, I was too quick :)

**Pull request merged**